playbooks/openshift-glusterfs/private/registry.yml

```diff
@@ -5,9 +5,12 @@
   hosts: oo_first_master
   roles:
   - role: openshift_persistent_volumes
-    when: openshift_hosted_registry_storage_glusterfs_swap | default(False)
+    when:
+    - openshift_hosted_registry_storage_kind | default(none) == 'glusterfs' or openshift_hosted_registry_storage_glusterfs_swap
+
+- import_playbook: ../../openshift_hosted/private/openshift_hosted_registry.yml
+
+- import_playbook: ../../openshift_hosted/private/openshift_hosted_wait_for_pods.yml
+
+- import_playbook: ../../openshift_hosted/private/openshift_hosted_registry_storage.yml
 
```

Comment:
Looks like there is an errant blank line at the end of the file.  Please fix for lint tests.